floki.yaml

```diff
@@ -0,0 +1,10 @@
+image:
```

Comment:
Great idea! Please could you add some documentation on how to use floki (and a link to it) to the README?

Reply:
Yes, will do.